libs/designsystem/card/src/card.component.ts

```diff
@@ -45,9 +50,27 @@ export class CardComponent implements OnInit, OnDestroy {
     this.sizesSortedByBreakpoint = this.sortSizesByBreakpoint(value);
   }
 
-  @HostBinding('class.flat')
+  /**
+   * @deprecated Using flat is no longer recomended as it is now part of the "variant" option. Will be removed in version 10.0.
+   */
   @Input()
-  flat: boolean = false;
+  set flat(isFlat: boolean) {
+    this._flat = isFlat;
+    console.warn(KIRBY_CARD_FLAT_PROPERTY_DEPRECATION_WARNING);
+  }
+
+  get flat(): boolean {
+    return this._flat;
+  }
+
+  @Input()
+  variant: 'elevated' | 'flat' | 'outlined' = 'elevated';
```

Comment:
Consider keeping naming in sync with UX:
```suggestion
  variant: 'elevated' | 'flat' | 'outline' = 'elevated';
```